src/sass/style.scss

```diff
@@ -16,88 +16,112 @@
 	font-weight: bold;
 }
 
-.buttonContainer {
-	@include size(25px, 1000px)
-    margin: auto;
+.error, .emptyUserTimeline {
+	color: red;
+	text-align: center;
+	padding: 10px 0px;
+	border-color: black;
+}
 
-	button{
-			float: right;
-		    top: 15px;
+.homeTimelineContainer {
+	width: 55%;
+	float: left;
+	.homeTimelineButton {
+		position: absolute;
+		left: 0px;
+		top: 100px
+	}
+	#homeTimelinePlaceholder {
+		.tweet {
+			&:nth-child(odd) {
+					background-color: #e8f5fd;
+			}
+			&:nth-child(even) {
+					background-color: #e9e9e9;
+			}
 		}
+	}
 }
 
-
-.error {
-		color: red;
-		text-align: center;
-		padding: 10px 0px;
-		border-color: black;
+.userTimelineContainer {
+	width: 45%;
+	float: right;
+	.userTimelineButton {
+		position: absolute;
+		right: 33px;
+		top: 100px;
 	}
-
-#timelinePlaceholder {
-	width: 1000px;
-	margin: 10px auto auto;
-
-	border: {
-		style: solid;
-		width: 1px;
+	#userTimelinePlaceholder {
+		.tweet {
+			&:nth-child(odd) {
+					background-color: #f5f8fa;
```

Comment:
ditto

Reply:
also I neglected to notice this but please don't hardcode the background color... have your tweets share the same css and override the color somehow.